Common/Platform/x86x64/Linux/PlatformDefines.h

```diff
@@ -37,6 +37,50 @@ using tchar = char;
 using tstring = std::string;
 #endif
 
+#ifdef __linux__
+#define WINAPIV
+#elif _WIN32
+#define WINAPIV __cdecl
```

Comment:
I think the marco named should be
1. LINUX
2. _LINUX
Don't define more different naming
Please look for all our available naming marco